notebooks/en/benchmarking_tgi.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>Take out "Here we can see the different settings for TGI Benchmark." and do:</p><p> </p><p>"Here are some of the more important TGI Benchmark settings."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>